### ISSUE TITLE
Any idea on how to display properly SQL results from Klipse ?

### ISSUE THREAD

**Issue by bhugueney** [timothypratley/asciidoctor-revealjs-klipse#6]
Thanks for this very interesting software : I love the ability to have interactive slides generated automatically !
Kudos for making Klipse working with reveal.js without having to resort to iframes ([the trick used by org-reveal](https://github.com/yjwen/org-reveal/blob/master/ox-reveal.el#L989) , which unfortunately break shared environment between code blocks).
I have been trying to enable interactive SQL (I'll be teaching some soon) and 

[resulting html table is displayed as text html](https://bhugueney.github.io/asciidoctor-revealjs-klipse/slides.html#/_sql_23_2)
while I was aiming for [something like this](http://blog.klipse.tech/javascript/2016/11/07/best-sql-tutorial.html) .
Do you have any insight on how to make it work ?

Thanks !

**Comment by timothypratley** [comment]
Hi @bhugueney 

If you add `editor_type: 'html'` to your window.klipse_settings,
then it will render the output table.

window.klipse_settings is in the file `docs/docinfo-footer.html`

Like this:

```
   window.klipse_settings = {
        selector: '.eval-clojure',
        selector_eval_js: '.eval-javascript',
        selector_eval_html: '.eval-html',
        selector_sql: '.eval-sql',
        editor_type: 'html'
    };
```

Mixing SQL and other languages will not work because there is only one option to govern the output.

Sorry for my slow response.

@viebel There doesn't seem to be any documentation on the SQL feature but I was able to find your working blog post example and see the `editor_type` was set there. It might be nice to advertise how to use SQL on the main readme... I didn't even know it was a feature, very cool! :)

**Comment by viebel** [comment]
@timothypratley 

I've just released a new version of Klipse - 7.9.2 - in which SQL Klipse snippets are automatically in html mode. No need for window.klipse_settings.editor_type = "html"; any more!
And there is a CSS file for nice table rendering: See https://github.com/viebel/klipse#sql

Code changes at: https://github.com/viebel/klipse/commit/1f84e13a98d0900f1524605c72668ff698206ffc

**Comment by timothypratley** [comment]
awesome :) Thank you :)
